examples/bridge-app/linux/main.cpp

```diff
@@ -229,11 +218,11 @@ void HandleDeviceStatusChanged(Device * dev, Device::Changed_t itemChangedMask)
 
     if (itemChangedMask & Device::kChanged_Name)
     {
-        uint8_t zclName[kUserLabelSize];
-        ToZclCharString(zclName, dev->GetName(), kUserLabelSize - 1);
+        MutableByteSpan zclName = MutableByteSpan();
```

Comment:
Was this tested?  This is going to create a span with a null buffer pointer, and then the implementation of `MakeZclCharString` in this PR will crash, while one that correctly checks lengths will not copy the data correctly....  This should probably be more like this:
```suggestion
        uint8_t zclName[kUserLabelSize];
        MutableByteSpan zclNameSpan(zclName);
```
and then pass that to MakeZclCharString.

Reply:
No, it slipped as I was updating the code. 
Updated now per suggestion.

Reply:
I also relied on the CI to tell me if it's broken as I've never run/tested this app.